<?xml version="1.0"?>

<launch>
  <arg name="goby_config" default="$(find alpha_acomms)/config/goby.yaml"/>
  <arg name="driver_config" default="$(find alpha_acomms)/config/evologics.yaml"/>

  <node pkg="alpha_acomms" type="modem_node" name="modem_node" output="screen">
    <rosparam command="load" file="$(arg goby_config)"/>
    <rosparam command="load" file="$(arg driver_config)"/>
    
    <param name="type" value="usbl"/>

  </node>

  <node pkg="alpha_acomms" type="alpha_acomms_node" name="alpha_acomms_node" output="screen">
    <rosparam command="load" file="$(arg goby_config)"/> 
    <param name="type" value="usbl"/>

  </node>
  

</launch>